src/rlx_mustache.erl

```diff
@@ -0,0 +1,230 @@
+%% The MIT License
+%%
+%% Copyright (c) 2009 Tom Preston-Werner <tom@mojombo.com>
```

Comment:
Doesn't that answer your question? ;)

It is the same module rebar has been using for templates for years.